.github/workflows/build-verification.yml

```diff
@@ -4,8 +4,13 @@ on:
   push:
     branches: [ main, citest ]
   pull_request:
-    branches: [ main ]
   workflow_dispatch:
+      inputs:
```

Comment:
do we still need this?

Reply:
Unless we decided to keep it on at all times, for staging/release builds as well, then yes, this is needed.

Reply:
got it!

Reply:
its meant to disable pts when triggered from workflow dispatch.